pulp_container/app/registry.py

```diff
@@ -231,6 +229,11 @@ async def dispatch_tag(self, request, tag, response_headers):
                 streamed back to the client.
 
         """
+        # return 404 in case the client is requesting docker manifest v2 schema 1
```

Comment:
I think this part is not required inside `dispatch_tag`. How about we remove the code and add the highlighted comment just above the `raise PathNotResolved` call placed at the end of the `get_tag` method? So, we will not forget about the rationale behind raising the exception.

Reply:
> I think this part is not required inside dispatch_tag.

Agreed. I added this part when I was having issues with the test_05_different_headers and setting "_wrong headers_". I'm rerunning the tests without this modification in dispatch_tag and will check if the tests will pass.

> How about we remove the code and add the highlighted comment just above the raise PathNotResolved call placed at the end of the get_tag method? So, we will not forget about the rationale behind raising the exception.

Good idea!